.github/workflows/stale_issue.yaml

```diff
@@ -0,0 +1,55 @@
+name: "Close stale issues"
+
+# Controls when the action will run.
+on:
+  schedule:
+  - cron: "0 0/3 * * *"
+
+jobs:
+  cleanup:
+    name: Stale issue job
+    runs-on: ubuntu-latest
+    steps:
+    - uses: aws-actions/stale-issue-cleanup@v4
+      with:
+        # Setting messages to an empty string will cause the automation to skip
+        # that category
+        ancient-issue-message: This is a very old issue that is probably not getting as much
+          attention as it deserves. We encourage you to check if this is still an issue in
+          the latest release and if you find that this is still a problem, please feel free
+          to provide a comment or open a new issue.
+        stale-issue-message: It looks like this issue has not been active for more than 5 days.
+          In the absence of more information, we will be closing this issue soon. If you find
+          that this is still a problem, please add a comment to prevent automatic closure, or
+          if the issue is already closed please feel free to reopen it.
+        stale-pr-message: It looks like this PR has not been active for more than five days. In
+          the absence of more information, we will be closing this PR soon. Please add a
+          comment to prevent automatic closure, or if the PR is already closed please feel
+          free to open a new one.
+
+        # These labels are required
+        stale-issue-label: closing-soon
+        exempt-issue-labels: no-auto-closure,feature-request
```

Comment:
The **no-auto-closure** label didn't exist so I created it. Is there a script to create the other labels we expect for this repo?